index.bs

```diff
@@ -1017,6 +1017,20 @@ For example,
 is more readable than
 <code highlight="js">window.scrollBy(50, 0)</code>.
 
+<h3 id="constructors">Make non-critical function parameters optional</h3>
+
+Parameters that are not strictly necessary should be made optional.
+Making arguments optional with reasonable defaults reduce overhead when the function is used,
+and increases user-friendliness.
+
+Optional parameters should treat empty or undefined as equal to the omission of the parameter.
+For example, given a function foo() which takes a single optional parameter,
+foo(undefined) and foo() should be equal,
```

Comment:
Maybe worth mentioning that WebIDL does this for you unless overloading is involved?  (At least I think that's what @annevk said in #73.)

Reply:
That's a good point, I'll make an edit to reflect this.

Reply:
Addressed in 0f0ae